examples/performance/stress-test/app.js

```diff
@@ -0,0 +1,293 @@
+import GL from '@luma.gl/constants';
+import {
+  AnimationLoop,
+  setParameters,
+  clear,
+  Texture2D,
+  Buffer,
+  Model,
+  CubeGeometry
+} from '@luma.gl/core';
+import {Matrix4, radians} from 'math.gl';
+/* eslint-disable spaced-comment, max-depth */
+/* global document, window */
+
+/*
+  Based on: https://github.com/tsherif/picogl.js/blob/master/examples/dof.html
+  Original algorithm: http://www.nutty.ca/?page_id=352&link=depth_of_field
+*/
+
+const INFO_HTML = `
+<p>
+  <b>Stress Test</b>.
+<p>
+`;
+
+const NUM_LAYERS = 250;
+const LAYER_DIM = 4;
+const LAYER_AREA = LAYER_DIM * LAYER_DIM;
+const NUM_DRAWS = NUM_LAYERS * LAYER_AREA;
+const NUM_ROWS = 10;
+const CUBES_PER_ROW = 10;
+const NUM_CUBES = CUBES_PER_ROW * NUM_ROWS;
+const NEAR = 100;
+const FAR = 1000.0;
+
+const stats = document.createElement('div');
+stats.innerHTML = `Drawing ${NUM_CUBES * NUM_DRAWS} cubes in ${NUM_DRAWS} draw calls`;
+stats.style.position = 'absolute';
+stats.style.top = '10px';
+stats.style.left = '10px';
+stats.style.color = 'white';
+
+const cpuStats = document.createElement('div');
+const gpuStats = document.createElement('div');
+stats.appendChild(cpuStats);
+stats.appendChild(gpuStats);
+document.body.appendChild(stats);
+
+class InstancedCube extends Model {
+  constructor(gl, props) {
+    const count = props.count;
+    const xforms = new Array(count);
+    const matrices = new Float32Array(count * 16);
+    const matrixBuffer = new Buffer(gl, matrices.byteLength);
+
+    const vs = `\
```

Comment:
Nit: Recommend defining these as global constants above the constructor, to make the JS code here more readable.